<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- saved from url=(0110)http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/Page_Config/content.html#literature -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=GBK">

<title>示范 蛋白iTRAQ定量分析 结题报告</title>
<link href="./report.css" rel="stylesheet" type="text/css">

<style type="text/css">
body{
        overflow:visible;
		position:relative;

	background-color:#FFFFFF;
	}
.note {color: #f00;}
</style>

</head>

<body>

<!-- ------------------------------------------------------------------------------------------------------------ -->


<div class="seq_cn1">

<!--  项目概述  -->
<div class="h0"><a name="ziti"></a><li>一、项目概述</li></div>

<br>
<div style="margin:0 auto; width:900px;">
<table align="center" style="width:900px;">
<tbody><tr><td>项目名称</td><td><b>GDP0000-Litchi_chinensis-8-iTraq</b></td></tr>
<tr><td>样品信息</td><td><b>CK1&nbsp;&#8260;&nbsp;CK2&nbsp;&#8260;&nbsp;UF1&nbsp;&#8260;&nbsp;UF2&nbsp;&#8260;&nbsp;FM1&nbsp;&#8260;&nbsp;FM2&nbsp;&#8260;&nbsp;MA1&nbsp;&#8260;&nbsp;MA2</b></td></tr>
<tr><td>样品间差异方案</td><td><b>CK1&nbsp;<font color="red">&#8596;</font>&nbsp;CK2&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;CK1&nbsp;<font color="red">&#8596;</font>&nbsp;FM1&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;CK1&nbsp;<font color="red">&#8596;</font>&nbsp;FM2&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;CK1&nbsp;<font color="red">&#8596;</font>&nbsp;MA1&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;CK1&nbsp;<font color="red">&#8596;</font>&nbsp;MA2&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;CK1&nbsp;<font color="red">&#8596;</font>&nbsp;UF1&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;CK1&nbsp;<font color="red">&#8596;</font>&nbsp;UF2</b></td></tr>
<tr><td>分组方案</td><td><b><font color="red">CK</font> : CK1 &amp; CK2&nbsp;&nbsp;&nbsp;<font color="#228B22">|</font>&nbsp;&nbsp;&nbsp;<font color="red">MA</font> : MA1 &amp; MA2&nbsp;&nbsp;&nbsp;<font color="#228B22">|</font>&nbsp;&nbsp;&nbsp;<font color="red">FM</font> : FM1 &amp; FM2&nbsp;&nbsp;&nbsp;<font color="#228B22">|</font>&nbsp;&nbsp;&nbsp;<font color="red">UF</font> : UF1 &amp; UF2</b></td></tr>
<tr><td>分组间差异方案</td><td><b>CK&nbsp;<font color="red">&#8596;</font>&nbsp;MA&nbsp;&nbsp;&nbsp;<font color="#228B22">|</font>&nbsp;&nbsp;&nbsp;CK&nbsp;<font color="red">&#8596;</font>&nbsp;FM&nbsp;&nbsp;&nbsp;<font color="#228B22">|</font>&nbsp;&nbsp;&nbsp;CK&nbsp;<font color="red">&#8596;</font>&nbsp;UF&nbsp;&nbsp;&nbsp;<font color="#228B22">|</font>&nbsp;&nbsp;&nbsp;FM&nbsp;<font color="red">&#8596;</font>&nbsp;UF</b></td></tr>
<tr><td>蛋白定量软件</td><td><b>Mascot 2.3.02</b></td></tr>
</tbody></table></div>
<br>
<hr>

<a name="expe"></a>
<p><br></p>
<div class="h0"><li>二、实验流程</li></div>
<br>

<a name="expe_1"></a>
<p><br></p>
<h2> 1）实验流程说明</h2>

<br>
<table style="border:0px #000000 solid;">
<tbody><tr style="border:0px #000000 solid;">
<td style="border:0px #000000 solid;">
<div class="tc">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="./flow.png" target="_blank"><img style="width:260px" src="./flow.png"></a></div>
<div class="tc">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>实验流程图</b></div>
</td>
<td style="border:0px #000000 solid; text-align:left">
同重同位素相对与绝对定量（Isobaric Tags for Relative and Absolute Quantitation，iTRAQ）技术<br>
是可以在一次实验中进行多达八个样品的蛋白质组定量技术，<br>
该定量方法几乎可以对任何蛋白样品进行定量分析，<br>
具有高定量精度的特点，目前已经越来越广泛的应用于定量蛋白质组学领域。<br>
iTRAQ定量蛋白质组技术自2004年在美国质谱年会第一次被提出以来，已经成为一种越来越广泛应用的定量蛋白质组技术。<br>
在2009年，有超过150篇的研究文章采用该技术,而且逐年增加。<br>
<br>
左图显示iTRAQ定量蛋白质组学实验的基本流程：<br>
第一步，从样品中提取蛋白。<br>
第二步，对提取后的蛋白样品进行还原烷基化处理，以便后续充分酶解蛋白。<br>
第三步，用GE公司的2D-quant-kit法进行蛋白质的浓度测定。<br>
第四步，等体积进行SDS（十二烷基磺酸钠）电泳。<br>
第五步，酶解蛋白。<br>
第六步，用iTRAQ试剂标记肽段。<br>
第七步，将标记后的肽段进行等量混合。<br>
第八步，对混合后的肽段使用强阳离子交换色谱(Strong Cation Exchange Choematography，SCX)进行预分离。<br>
第九步，进行液相串联质谱(Liquid Chromatography Coupled with Tandem Mass Spectrometry，LC-MS/MS)分析<br>
</td>
</tr>
</tbody></table>

<a name="expe_2"></a>
<p><br></p>
<h2> 2）iTRAQ定量原理</h2>
<br>

<table style="border:0px #000000 solid;">
<tbody><tr style="border:0px #000000 solid;">
<td style="border:0px #000000 solid;">
<div class="tc">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="./method.png" target="_blank"><img style="width:300px" src="./method.png"></a></div>
<div class="tc">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>iTRAQ试剂结构</b></div>
</td>
<td style="border:0px #000000 solid; text-align:left">
iTRAQ试剂由报告离子（Report Group），平衡基团（Balance Group），反应基团（Peptide Reactive Group）三部分构成，<br>
反应基团可以与肽段N-端或赖氨酸侧链发生反应，从而可以标记任何肽段。<br>
</td>
</tr>
</tbody></table>

<br><hr><br>

<table style="border:0px #000000 solid;">
<tbody><tr style="border:0px #000000 solid;">
<td style="border:0px #000000 solid;">
<div class="tc">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="./method1.jpg" target="_blank"><img style="width:420px" src="./method1.jpg"></a></div>
<div class="tc">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>iTRAQ定量蛋白质组学原理</b></div>
</td>
<td style="border:0px #000000 solid; text-align:left">
左图显示了iTRAQ定量技术的基本原理及主要步骤，<br>
iTRAQ定量方法可以在一次串联质谱实验中同时比较8个样品中的蛋白的相对含量，<br>
主要步骤如图中所示，分别为蛋白提取、酶解、标记、混合、SCX预分离、液相串联质谱分析。<br>
在一级质谱时，平衡基团可以确保无论用哪种报告离子标记肽段，都显示为相同的质荷比值。<br>
在二级质谱时，平衡基团发生中性丢失，而报告离子的强度则可以反映肽段的相对丰度值。<br>
图中最下方为一张MS/MS谱图，横坐标为子离子质荷比值，纵坐标为离子强度。<br>
8个彩色峰表示iTRAQ试剂的8种报告离子，其高度分别代表了肽段的相对含量，用于后续定量。<br>
其余黑色峰为肽段碎裂后的子离子峰，用于后续鉴定。<br>
</td>
</tr>
</tbody></table>

<br><hr><br>
&nbsp;&nbsp;&nbsp;&nbsp;利用iTRAQ技术进行蛋白质组定量的优势主要体现在：<br><br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(1)&nbsp;&nbsp;由于试剂可以标记任何肽段，包括翻译后修饰肽段，因此可以极大的提高蛋白鉴定的可信度和覆盖度。<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2)&nbsp;&nbsp;由于可以对一个蛋白的多个肽段进行定量，因此可以提高定量的可信度。<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(3)&nbsp;&nbsp;对于发现生物标记物，是一种高通量的研究方法。<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(4)&nbsp;&nbsp;定量精度较高。<br>
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(5)&nbsp;&nbsp;可以在一次实验中，进行多达8个样品的比较。利用基于同一个参考样品的办法，可以进行多于8个样品的定量比较。<br>

<br><hr>


<!--标准信息分析-->
<a name="analy"></a>
<p><br></p>
<div class="h0"><li>三、分析流程</li></div>
<br>
<table style="border:0px #000000 solid;">
<tbody><tr style="border:0px #000000 solid;">
<td style="border:0px #000000 solid;">
<div class="tc"><a href="./flow_a.png" target="_blank"><img style="width:600px" src="./flow_a.png"></a></div>
<div class="tc"><b>分析流程图</b></div>
</td>
<td style="border:0px #000000 solid; text-align:left">
左图显示iTRAQ定量蛋白质组学的基本信息分析流程：<br>
首先对于质谱下机的原始文件，进行峰识别，得到峰列表。<br>
其次建立参考数据库，进行肽段及蛋白质的鉴定。<br>
最后比较各蛋白在各样品之间的相对含量的关系，从而获得一些感兴趣的重要蛋白。<br>
也可以将转录组数据和蛋白组数据结合起来，进行蛋白组与转录组的关联分析。<br>
</td>
</tr>
</tbody></table>

<a name="raw_data"></a>
<p><br></p>
<h2> 1）原始质谱数据</h2>

<p><span class="bold">质谱文件需要转换成mgf格式后才能被使用，因此以mgf作为初始文件，mgf主要包含了二级质谱（MS/MS）谱图的信息，其基本格式如下：</span></p>
<pre>BEGIN IONS
TITLE=Spectrum1 scans: 2,
PEPMASS=588.84003 11629.05371
CHARGE=2+
RTINSECONDS=0
SCANS=2
115.55481 258.814
116.11040 496.234
136.94980 420.235
173.90498 377.256
180.86188 368.726
190.94720 352.431
………
END IONS
</pre>
<p><span class="bold">其中“BEGIN IONS”和“END IONS”是每一张谱的开始和结束位置。“TITLE”为该谱图的编号信息。</span></p>
<p><span class="bold">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;“PEPMASS”为母离子质荷比和强度值。“CHARGE”为母离子所带电荷数。</span></p>
<p><span class="bold">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;“RTINSECONDS”为保留时间。“SCANS”为扫描编号。其余各行为肽段经碎裂后的子离子的质荷比值和强度信息。</span></p>

<br><hr> <!-- ---------------------------------------------------------------------------------------------------------------------------- -->

<a name="db_choose"></a>
<p><br></p>
<h2> 2）数据库的选择</h2>

<p><span class="bold">数据库的选择是基于质谱数据的蛋白质鉴定策略中的重要一步，最终鉴定到的蛋白质序列都来源于被选择的数据库中。</span></p>
<p><span class="bold">目前使用到的数据库主要可以分为两类，一类是NCBI来维护的，另外一类是由EBI负责维护。数据库建立的方法主要有以下几种：</span></p>
<p><span class="bold">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;1.NCBInr全库；</span></p>
<p><span class="bold">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2.NCBInr分类库，包括动物全库、植物全库、微生物全库、细菌全库等；</span></p>
<p><span class="bold">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3.SwissProt/Uniprot/IPI分类库，包括动物全库、植物全库、微生物全库、细菌库等；</span></p>
<p><span class="bold">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;4.NCBInr对应物种库，包括人、玉米、对虾、大肠杆菌等；</span></p>
<p><span class="bold">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;5.其他物种库，如柑橘克里曼丁蛋白库。</span></p>
<p><span class="bold">在选择数据库时，遵循如下原则，若为已经测序生物，直接选用该物种数据库，若为非测序生物，则选择与被测样品最为相关的大类蛋白质组数据库。</span></p>

<br><hr> <!-- ---------------------------------------------------------------------------------------------------------------------------- -->

<a name="mascot"></a>
<p><br></p>
<h2> 3）Mascot搜索</h2>

<p><span class="bold">Mascot 是一个蛋白质鉴定软件，曾被 Frost&amp;Sullivan 研究机构评为生物质谱软件的黄金标准。</span></p>
<p><span class="bold">我们使用的软件版本为 Mascot 2.3.02。操作时需要将 mgf 文件提交至 Mascot，选择已经建立好的数据库，然后进行数据库搜索。</span></p>
<br> <!-- ---------------------------------------------------------------------------------------------------------------------------- -->

<div class="tc"><b>Mascot 搜索参数信息表</b></div>
<div style="margin:0 auto; width:900px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>Item</th><th>Value</th></tr>
<tr><td>Type of search</td><td>MS/MS Ion search</td></tr>
<tr><td>Enzyme</td><td>Trypsin</td></tr>
<tr><td>Fragment Mass Tolerance</td><td>+/- 0.05 Da</td></tr>
<tr><td>Mass Values</td><td>Monoisotopic</td></tr>
<tr><td>Variable modifications</td><td>Gln-&gt;pyro-Glu (N-term Q),Oxidation (M)</td></tr>
<tr><td>Peptide Mass Tolerance</td><td>20 ppm</td></tr>
<tr><td>Instrument type</td><td>Triple TOF 5600</td></tr>
<tr><td>Max Missed Cleavages</td><td>1</td></tr>
<tr><td>Fixed modifications</td><td>Carbamidomethyl (C),iTRAQ8plex (N-term),iTRAQ8plex (K)</td></tr>
<tr><td>Protein Mass</td><td>Unrestricted</td></tr>
<tr><td>Database</td><td>mascot_db_20150923.fa (33209 sequences)</td></tr>
<tr><td colspan="2" style="border-bottom:2px #60EF60 solid; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60">&nbsp;</td></tr>
</tbody></table></div>

<br><hr> <!-- ---------------------------------------------------------------------------------------------------------------------------- -->

<a name="pro_identify"></a>
<p><br></p>
<h2> 4）蛋白质鉴定</h2>

<a name="basic_info"></a>
<p><br></p>
<h3> （1）基本鉴定信息 </h3>
<br>

<div style="margin:0 auto; width:1005px;"> <table style="border:0px #000000 solid;">
<tbody><tr style="border:0px #000000 solid;">

<td style="border:0px #000000 solid;"><div class="tc">
<a href="./basic_identi_stat.png" target="_blank"><img style="width:500px" src="./basic_identi_stat.png"></a>
</div></td>

<td style="border:0px #000000 solid;" rowspan="2"><div class="tc">
<div class="tc"><b>基本鉴定信息统计表</b></div>
<div style="margin:0 auto; width:500px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>Item</th><th>Value</th></tr>
<tr><td>Total Spectra</td><td>197103</td></tr>
<tr><td>Spectra</td><td>36997</td></tr>
<tr><td>Unique Specra</td><td>30173</td></tr>
<tr><td>Peptide</td><td>11670</td></tr>
<tr><td>Unique Peptide</td><td>10493</td></tr>
<tr><td>Protein</td><td>3467</td></tr>
<tr><td colspan="2" style="border-bottom:2px #60EF60 solid; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60">&nbsp;</td></tr>
</tbody></table></div>

</div></td>
</tr>

<tr><td><b>基本鉴定信息统计图</b></td></tr><tr> </tr></tbody></table> </div>

<p><span class="bold">上图横坐标为鉴定类别，纵坐标为数量。Total Spectra为二级谱图总数，Spectra为匹配到的谱图数量，</span></p>
<p><span class="bold">Unique Spectra为匹配到特有肽段的谱图数量，Peptide为鉴定到的肽段的数量，Unique Peptide为鉴定到特有肽段序列的数量，Protein为鉴定到的蛋白质数量。</span></p>

<br>

<a name="pro_xfz"></a>
<p><br></p>
<h3> （2）蛋白质相对分子质量 </h3>
<br>

<div style="margin:0 auto; width:1005px;"> <table style="border:0px #000000 solid;">
<tbody><tr style="border:0px #000000 solid;">

<td style="border:0px #000000 solid;"><div class="tc">
<a href="./protein_mass.png" target="_blank"><img style="width:500px" src="./protein_mass.png"></a>
</div></td>

<td style="border:0px #000000 solid;" rowspan="2"><div class="tc">
<div class="tc"><b>蛋白质质量分布表</b></div>
<div style="margin:0 auto; width:500px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>MW(in kDa)</th><th>Number of proteins</th></tr>
<tr><td>0-10</td><td>54</td></tr>
<tr><td>10-20</td><td>403</td></tr>
<tr><td>20-30</td><td>518</td></tr>
<tr><td>30-40</td><td>516</td></tr>
<tr><td>40-50</td><td>489</td></tr>
<tr><td>50-60</td><td>398</td></tr>
<tr><td>60-70</td><td>284</td></tr>
<tr><td>70-80</td><td>222</td></tr>
<tr><td>80-90</td><td>130</td></tr>
<tr><td>90-100</td><td>105</td></tr>
<tr><td>&gt;100</td><td>348</td></tr>
<tr><td colspan="2" style="border-bottom:2px #60EF60 solid; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60">第一列 MW（in.kDa）为蛋白质分子质量范围，<br>第二列（Number of proteins in class）为对应蛋白质分子质量范围内的蛋白质数量</td></tr>
</tbody></table></div>
</div></td>
</tr>

<tr><td><b>蛋白质的质量分布图</b></td></tr><tr> </tr></tbody></table> </div>

<p><span class="bold">上图横坐标为鉴定到的蛋白质分子质量（单位：千道尔顿，kDa），纵坐标为鉴定到的蛋白质数量。</span></p>

<br>

<a name="pro_len"></a>
<p><br></p>
<h3> （3）肽段序列长度分布 </h3>
<br>

<div style="margin:0 auto; width:805px;"> <table style="border:0px #000000 solid;"> <tbody><tr style="border:0px #000000 solid;"> <td style="border:0px #000000 solid;"><div class="tc">
<a href="./peptide_len.png" target="_blank"><img style="width:500px" src="./peptide_len.png"></a>
</div></td> </tr> <tr><td><b>肽段长度分布图</b></td></tr><tr> </tr></tbody></table> </div>

<p><span class="bold">上图表示不同长度肽段占所有肽段的百分比。横坐标为肽段氨基酸残基数，纵坐标为该长度肽段占所有肽段的百分比。</span></p>

<br>

<a name="pro_cov"></a>
<p><br></p>
<h3> （4）肽段序列覆盖度 </h3>
<br>

<div style="margin:0 auto; width:1005px;"> <table style="border:0px #000000 solid;">
<tbody><tr style="border:0px #000000 solid;">

<td style="border:0px #000000 solid;"><div class="tc">
<a href="./protein_cov.png" target="_blank"><img style="width:500px" src="./protein_cov.png"></a>
</div></td>

<td style="border:0px #000000 solid;" rowspan="2"><div class="tc">
<div class="tc"><b>肽段序列覆盖度分布表</b></div>
<div style="margin:0 auto; width:500px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>Sequence Coverage(%)</th><th>Number of proteins</th></tr>
<tr><td>&lt;=5%</td><td>984</td></tr>
<tr><td>&lt;=10%</td><td>826</td></tr>
<tr><td>&lt;=15%</td><td>548</td></tr>
<tr><td>&lt;=20%</td><td>335</td></tr>
<tr><td>&lt;=25%</td><td>227</td></tr>
<tr><td>&lt;=30%</td><td>191</td></tr>
<tr><td>&lt;=35%</td><td>129</td></tr>
<tr><td>&lt;=40%</td><td>84</td></tr>
<tr><td>&lt;=100%</td><td>143</td></tr>
<tr><td>Total</td><td>3467</td></tr>
<tr><td colspan="2" style="border-bottom:2px #60EF60 solid; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60">第一列 Sequence Coverage(%)为蛋白质肽段覆盖度的范围，<br>第二列 Number of proteins in  class 为这一覆盖范围内的蛋白质数量</td></tr>
</tbody></table></div>
</div></td>
</tr>

<tr><td><b>肽段序列覆盖度分布图</b></td></tr><tr> </tr></tbody></table> </div>

<p><span class="bold">上图显示不同覆盖度的蛋白比例，不同颜色代表不同的序列覆盖度范围， 饼状图百分比显示了处于不同覆盖度范围的蛋白数量占总蛋白数量的比例。</span></p>

<br>

<a name="pro_num"></a>
<p><br></p>
<h3> （5）鉴定肽段数量分布 </h3>
<br>

<div style="margin:0 auto; width:805px;"> <table style="border:0px #000000 solid;"> <tbody><tr style="border:0px #000000 solid;"> <td style="border:0px #000000 solid;"><div class="tc">
<a href="./peptide_number.png" target="_blank"><img style="width:500px" src="./peptide_number.png"></a>
</div></td> </tr> <tr><td><b>鉴定肽段数量分布图</b></td></tr><tr> </tr></tbody></table> </div>

<p><span class="bold">上图显示鉴定到的蛋白所含肽段的数量分布情况，横坐标为鉴定蛋白的肽段数量范围，数量随着匹配肽段数量的增加而减少。</span></p>

<br>

<p><span class="bold">鉴定结果见 <a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/1_Identification/" target="_blank">1_Identification</a> 文件夹下 <a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/1_Identification/Litchi_chinensis-8-iTraq.detail.xls" target="_blank">Litchi_chinensis-8-iTraq.detail.xls</a> 与 <a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/1_Identification/Litchi_chinensis-8-iTraq.overall.xls" target="_blank">Litchi_chinensis-8-iTraq.overall.xls</a> 文件。</span></p>
<p><span class="bold">其中sample_detail.xls中包含了肽段和蛋白的信息，sample_overall.xls 中包含了蛋白的信息。</span></p>

<br><hr> <!-- ---------------------------------------------------------------------------------------------------------------------------- -->
<a name="pro_annot"></a>
<p><br></p>
<h2> 5）蛋白质注释</h2>

<a name="path_anno"></a>
<p><br></p>
<h3> （1）Pathway代谢通路注释 </h3>
<br>
<p><span class="bold">在生物体内，不同蛋白相互协调行使其生物学行为，基于 Pathway 的分析有助于更进一步了解其生物学功能。KEGG是有关 Pathway 的主要公共数据库（Kanehisa，2008），通过 Pathway 分析能确定蛋白质参与的最主要生化代谢途径和信号转导途径。</span></p>

<ul><li>&nbsp;&nbsp;&nbsp;<font size="4px" color="#8B008B"><b>&#9674;</b></font>&nbsp;<b>Pathway 分析结果报告</b> : &nbsp;&nbsp;
<a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/1_KEGG/Litchi_chinensis-8-iTraq.htm" target="_blank">Litchi_chinensis-8-iTraq.htm</a>&nbsp;&nbsp;&nbsp;表头各列含义请参考&nbsp;&nbsp;:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/Page_Config/content.html#help_path_htm" target="_blank" title="Pathway分析结果帮助文档"><img src="./need_help_2.png" style="width: 20px;"></a>
</li></ul>
<br>

<ul><li>&nbsp;&nbsp;&nbsp;<font size="4px" color="#8B008B"><b>&#9674;</b></font>&nbsp;<b>基因 ID 及对应 KO 号列表</b> : &nbsp;&nbsp;
<a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/1_KEGG/Litchi_chinensis-8-iTraq.ko" target="_blank">Litchi_chinensis-8-iTraq.ko</a>
</li></ul>
<br>

<ul><li>&nbsp;&nbsp;&nbsp;<font size="4px" color="#8B008B"><b>&#9674;</b></font>&nbsp;<b>基因的 Pathway 列表</b> : &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
<a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/1_KEGG/Litchi_chinensis-8-iTraq.path.xls" target="_blank">Litchi_chinensis-8-iTraq.path.xls</a>
</li></ul>

<br>
<div class="tc"><font color="#218868" size="3px"><b>基因的 Pathway 列表（只展示前五行）</b></font></div>
<div style="margin:0 auto; width:1100px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>KEGG_A_class</th><th>KEGG_B_class</th><th>#Pathway</th><th>Count (1875)</th><th>Pathway ID</th><th>Genes</th><th>KOs</th></tr>
<tr><td>Metabolism</td><td>Global and  ...</td><td>Metabolic p ...</td><td>644</td><td>ko01100</td><td>Unigene0000291;Unige ...</td><td>K00064+K02705+K07964+ ...</td></tr>
<tr><td>Metabolism</td><td>Global and  ...</td><td>Biosynthesi ...</td><td>361</td><td>ko01110</td><td>Unigene0000291;Unige ...</td><td>K00064+K00026+K01900+ ...</td></tr>
<tr><td>Genetic Inf ...</td><td>Translation</td><td>Ribosome</td><td>117</td><td>ko03010</td><td>Unigene0001120;Unige ...</td><td>K02966+K02915+K02900+ ...</td></tr>
<tr><td>Genetic Inf ...</td><td>Translation</td><td>RNA transpo ...</td><td>80</td><td>ko03013</td><td>Unigene0000631;Unige ...</td><td>K03260+K14306+K03248+ ...</td></tr>
<tr style="border-bottom:2px #AAAAAA solid;"><td>Genetic Inf ...</td><td>Folding, so ...</td><td>Protein pro ...</td><td>79</td><td>ko04141</td><td>Unigene0001956;Unige ...</td><td>K14006+K09540+K07953+ ...</td></tr>
<tr><td colspan="7" style="border-bottom:solid #AAAAAA 2px; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60">表头各列含义请参考&nbsp;&nbsp;:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/Page_Config/content.html#help_pathway_list" target="_blank" title="基因Pathway表帮助文档"><img src="./need_help_2.png" style="width: 20px;"></a></td></tr>
</tbody></table></div>

<br>


<a name="go_ana"></a>
<p><br></p>
<h3> （2）GO分析 </h3>
<br>

<p><span class="bold">Gene Ontology（简称GO）是一个国际标准化的基因功能分类体系，提供了一套动态更新的标准词汇表（Controlled Vocabulary）来全面描述生物体中基因和基因产物的属性。GO总共有三个本体（Ontology），分别描述基因的分子功能（Molecular Function）、所处的细胞位置（Cellular Component）、参与的生物过程（Biological Process）。</span></p>
<br>
<p><span class="bold">详细信息见网站：<a href="http://www.geneontology.org/" target="_blank">http://www.geneontology.org</a></span></p>
<br>
<p><span class="bold">我们针对鉴定出的所有蛋白进行GO功能注释分析，给出的结果包括两部分：protein2go和go2protein。</span></p>
<p><span class="bold">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;protein2go：针对每个蛋白，给出所有相应的GO功能的ID列表</span></p>
<p><span class="bold">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;go2protein：针对三个ontology（cellular component，biological process，molecular function）中所涉及的GO 条目，</span></p>
<p><span class="bold">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;列出所有相应的蛋白的ID及蛋白个数，同时作出统计图，略去没有相应蛋白的GO 条目。</span></p>
<br>

<div style="margin:0 auto; width:805px;"> <table style="border:0px #000000 solid;">

<tbody><tr style="border:0px #000000 solid;"> <td style="border:0px #000000 solid;"><div class="tc">
<a href="./Litchi_chinensis-8-iTraq.go.png" target="_blank"><img style="width:1000px" src="./Litchi_chinensis-8-iTraq.go.png"></a>
</div></td> </tr>

<tr><td><b>所有蛋白的GO分类图</b></td></tr><tr> </tr></tbody></table> </div>

<p><span class="bold">GO分类图显示了三个本体中所涉及到各条目的分布情况。</span></p>
<br><br>

<ul><li>&nbsp;&nbsp;&nbsp;<font size="4px" color="#8B008B"><b>&#9674;</b></font>&nbsp;<b>GO 注释表</b> : &nbsp;&nbsp;
<a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/2_GO/Litchi_chinensis-8-iTraq.wego.xls" target="_blank">Litchi_chinensis-8-iTraq.wego.xls</a>
</li></ul>

<div class="tc"><font color="#218868" size="3px"><b>GO 注释表（只展示前五行）</b></font></div>
<div style="margin:0 auto; width:800px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>Protein ID</th><th>GO ID</th><th>&nbsp;</th><th>&nbsp;</th><th>&nbsp;</th><th>&nbsp;</th></tr>

<tr><td>Unigene0000194</td><td>GO:0044424</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td>Unigene0000198</td><td>GO:0043231</td><td>GO:0016616</td><td>GO:0044710</td><td>GO:0000166</td><td>&nbsp;</td></tr>
<tr><td>Unigene0000291</td><td>GO:0016646</td><td>GO:0019852</td><td>GO:0016639</td><td>GO:0044444</td><td>...</td></tr>
<tr><td>Unigene0000293</td><td>GO:0016811</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr style="border-bottom:2px #AAAAAA solid;"><td>Unigene0000389</td><td>GO:0019787</td><td>GO:0032446</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr>
<tr><td colspan="6" style="border-bottom:solid #AAAAAA 2px; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60">&nbsp;</td></tr>
</tbody></table></div>

<br><br>
<ul><li>&nbsp;&nbsp;&nbsp;<font size="4px" color="#8B008B"><b>&#9674;</b></font>&nbsp;<b>GO 分类表</b> : &nbsp;&nbsp;
<a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/2_GO/Litchi_chinensis-8-iTraq.go.xls" target="_blank">Litchi_chinensis-8-iTraq.go.xls</a>
</li> </ul>

<div class="tc"><font color="#218868" size="3px"><b>GO 分类表（只展示前五行）</b></font></div>
<div style="margin:0 auto; width:1100px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>Ontology</th><th>Class</th><th>number_of_uniprot</th><th>proteins_of_uniprot</th></tr>
<tr><td>biological_process</td><td>metabolic process</td><td>1806</td><td>Unigene0043292;Unigene0016182; ......</td></tr>
<tr><td>biological_process</td><td>cellular process</td><td>1521</td><td>Unigene0047599;Unigene0043292; ......</td></tr>
<tr><td>biological_process</td><td>single-organism process</td><td>1332</td><td>Unigene0047599;Unigene0010328; ......</td></tr>
<tr><td>biological_process</td><td>response to stimulus</td><td>511</td><td>Unigene0047599;Unigene0000626; ......</td></tr>
<tr style="border-bottom:2px #AAAAAA solid;"><td>biological_process</td><td>biological regulation</td><td>483</td><td>Unigene0047599;Unigene0054733; ......</td></tr>
<tr><td colspan="4" style="border-bottom:solid #AAAAAA 2px; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60">表头各列含义请参考&nbsp;&nbsp;:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/Page_Config/content.html#help_go_xls" target="_blank" title="GO 分类表帮助文档"><img src="./need_help_2.png" style="width: 20px;"></a></td></tr>
</tbody></table></div>

<a name="cog_anno"></a>
<p><br></p>
<h3> 3）COG/KOG注释</h3>
<br>
<p><span class="bold">COG（Cluster of Orthologous Groups of Proteins 蛋白相邻类的聚簇）是对蛋白质进行直系同源分类的数据库。构成每个 COG 的蛋白都是被假定为来自于一个祖先蛋白，并且因此或者是 Orthologs 或者是 Paralogs。Orthologs 是指来自于不同物种的由垂直家系（物种形成）进化而来的蛋白，并且典型的保留与原始蛋白有相同的功能。Paralogs 是那些在一定物种中的来源于基因复制的蛋白，可能会进化出新的与原来有关的功能。我们将鉴定到的和 COG 数据库进行比对，预测这些蛋白可能的功能并对其做功能分类统计。</span></p>
<p>原核生物用 COG 注释，真核生物用 KOG 注释</p>
<table style="border:0px #000000 solid;"> <tbody><tr style="border:0px #000000 solid;"> <td style="border:0px #000000 solid;">
<a href="./Litchi_chinensis-8-iTraq.kog.png" target="_blank"><img style="width:50%" src="./Litchi_chinensis-8-iTraq.kog.png"></a>
</td> </tr>
<tr style="border:0px #000000 solid;">
<td style="border:0px #000000 solid;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<font color="#218868" size="2.5px"><b>kog 功能分类图</b></font></td>
</tr> </tbody></table>

<ul><li>&nbsp;&nbsp;&nbsp;<font size="4px" color="#8B008B"><b>&#9674;</b></font>&nbsp;<b>kog 分类表</b> : &nbsp;&nbsp;
<a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/3_COG_KOG/Litchi_chinensis-8-iTraq.kog.class.annot.xls" target="_blank">Litchi_chinensis-8-iTraq.kog.class.annot.xls</a>
</li></ul>

<br>
<div class="tc"><font color="#218868" size="3px"><b>kog 分类表（只展示前五行）</b></font></div>
<div style="margin:0 auto; width:1000px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>Code</th><th>Functional-Categories</th><th>Protein-Number</th><th>Proteins</th></tr>
<tr><td>A</td><td>RNA processing and modification </td><td>403</td><td>Unigene0022121,KOG0213 Unigene0022231,KOG2191 ......</td></tr><tr><td>B</td><td>Chromatin structure and dynamics </td><td>81</td><td>Unigene0022035,KOG0355 Unigene0022734,KOG0264 ......</td></tr><tr><td>C</td><td>Energy production and conversion </td><td>322</td><td>Unigene0021479,KOG2454 Unigene0021479,KOG2450 ......</td></tr><tr><td>D</td><td>Cell cycle control, cell division, chromosome partitioning </td><td>96</td><td>Unigene0022128,KOG0005 Unigene0023724,KOG1126 ......</td></tr><tr style="border-bottom:2px #AAAAAA solid;"><td>E</td><td>Amino acid transport and metabolism </td><td>258</td><td>Unigene0021479,KOG2449 Unigene0022268,KOG4201 ......</td></tr><tr><td colspan="4" style="border-bottom:solid #AAAAAA 2px; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60">表头各列含义请参考&nbsp;&nbsp;:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/Page_Config/content.html#help_cog_class" target="_blank" title="kog 分类表帮助文档"><img src="./need_help_2.png" style="width: 20px;"></a></td></tr>
</tbody></table></div>

<br>
<ul><li>&nbsp;&nbsp;&nbsp;<font size="4px" color="#8B008B"><b>&#9674;</b></font>&nbsp;<b>kog 注释表</b> : &nbsp;&nbsp;
<a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/3_COG_KOG/Litchi_chinensis-8-iTraq.kog.gene.annot.xls" target="_blank">Litchi_chinensis-8-iTraq.kog.gene.annot.xls</a>
</li></ul>

<br>
<div class="tc"><font color="#218868" size="3px"><b>kog 注释表（只展示前五行）</b></font></div>
<div style="margin:0 auto; width:1100px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>Protein</th><th>Protein-or-Domain</th><th>Score</th><th>E-Value</th><th>COG/KOG-ID</th><th>Function-Description</th><th>Code</th><th>Functional-Categories</th></tr>
<tr><td>Unigene0021248</td><td>At5g13640</td><td>  542</td><td>2e-178</td><td>KOG2369</td><td>Lecithin:cholesterol  ...</td><td>I</td><td>Lipid transport and m ...</td></tr><tr><td>Unigene0021248</td><td>At3g44830</td><td>  412</td><td>6e-129</td><td>KOG2369</td><td>Lecithin:cholesterol  ...</td><td>I</td><td>Lipid transport and m ...</td></tr><tr><td>Unigene0021248</td><td>SPBC776.14</td><td>  147</td><td>3e-36</td><td>KOG2369</td><td>Lecithin:cholesterol  ...</td><td>I</td><td>Lipid transport and m ...</td></tr><tr><td>Unigene0021248</td><td>YNR008w</td><td>  135</td><td>4e-32</td><td>KOG2369</td><td>Lecithin:cholesterol  ...</td><td>I</td><td>Lipid transport and m ...</td></tr><tr><td>Unigene0021248</td><td>At1g04010</td><td>68.6</td><td>3e-11</td><td>KOG2369</td><td>Lecithin:cholesterol  ...</td><td>I</td><td>Lipid transport and m ...</td></tr><tr style="border-bottom:2px #AAAAAA solid;"><td>Unigene0021248</td><td>At1g04010</td><td>68.6</td><td>3e-11</td><td>KOG2369</td><td>Lecithin:cholesterol  ...</td><td>I</td><td>Lipid transport and m ...</td></tr><tr><td colspan="8" style="border-bottom:solid #AAAAAA 2px; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60">表头各列含义请参考&nbsp;&nbsp;:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/Page_Config/content.html#help_cog_annot" target="_blank" title="kog 注释表帮助文档"><img src="./need_help_2.png" style="width: 20px;"></a></td></tr>
</tbody></table></div>

<br>

<ul><li>&nbsp;&nbsp;&nbsp;<font size="4px" color="#8B008B"><b>&#9674;</b></font>&nbsp;<b>blast 比对 kog 结果表</b> : &nbsp;&nbsp;
<a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/3_COG_KOG/Litchi_chinensis-8-iTraq.blast.kog.xls" target="_blank">Litchi_chinensis-8-iTraq.blast.kog.xls</a>
<!--
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
<a href="#help_blast_result" target="_blank" title="blast 比对 kog 结果表说明文档"><img src="../Page_Config/images/need_help.png" style="width: 20px;" /></a>
-->
</li></ul>
<br><hr> <!-- ---------------------------------------------------------------------------------------------------------------------------- -->

<a name="pro_quali"></a>
<p><br></p>
<h2> 6）蛋白质定量</h2>

<a name="exp_info"></a>
<p><br></p>
<h3> （1）实验标记信息 </h3>
<br>

<div class="tc"><b>实验 itraq 标记信息表</b></div>
<div style="margin:0 auto; width:600px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>样品编号</th><th>标记编号</th></tr>
<tr><td>CK1</td><td>113</td></tr>
<tr><td>CK2</td><td>114</td></tr>
<tr><td>UF1</td><td>115</td></tr>
<tr><td>UF2</td><td>116</td></tr>
<tr><td>FM1</td><td>117</td></tr>
<tr><td>FM2</td><td>118</td></tr>
<tr><td>MA1</td><td>119</td></tr>
<tr><td>MA2</td><td>121</td></tr>
<tr><td colspan="2" style="border-bottom:2px #60EF60 solid; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60"></td></tr>
</tbody></table></div>

<a name="all_stat"></a>
<p><br></p>
<h3> （2）总体定量统计 </h3>
<br>
<ul><li>&nbsp;&nbsp;&nbsp;<font size="4px" color="#8B008B"><b>&#9674;</b></font>&nbsp;<b>总体蛋白定量统计注释表</b> : &nbsp;&nbsp;
<a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/1_Expression/All_Exp.annot.xls" target="_blank">All_Exp.annot.xls</a>
</li></ul>
<br>

<a name="diff_analy"></a>
<p><br></p>
<h2> 7）差异蛋白分析 </h2>
<br>

<a name="diff_stat"></a>
<p><br></p>
<h3> （1）差异蛋白统计 </h3>
<br>

<div style="margin:0 auto; width:1005px;"> <table style="border:0px #000000 solid;">
<tbody><tr style="border:0px #000000 solid;">

<td style="border:0px #000000 solid;"><div class="tc">
<a href="./sample_diff.png" target="_blank"><img style="width:500px" src="./sample_diff.png"></a>
</div></td>

<td style="border:0px #000000 solid;" rowspan="2"><div class="tc">
<div class="tc"><b>样品间差异蛋白统计表</b></div>
<div style="margin:0 auto; width:500px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>样品间比较</th><th>上调蛋白数量</th><th>下调蛋白数量</th></tr>
<tr><td>CK2/CK1</td><td>141</td><td>207</td></tr>
<tr><td>FM1/CK1</td><td>345</td><td>394</td></tr>
<tr><td>FM2/CK1</td><td>490</td><td>657</td></tr>
<tr><td>MA1/CK1</td><td>391</td><td>495</td></tr>
<tr><td>MA2/CK1</td><td>491</td><td>655</td></tr>
<tr><td>UF1/CK1</td><td>406</td><td>528</td></tr>
<tr><td>UF2/CK1</td><td>358</td><td>303</td></tr>
<tr><td colspan="3" style="border-bottom:2px #60EF60 solid; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60">依据蛋白质丰度水平，当差异倍数达到1.2倍以上，<br>且经统计检验其p-value值小于0.05时，视为差异蛋白。</td></tr>
</tbody></table></div>
</div></td>
</tr>

<tr><td><b>样品间差异蛋白数量统计图</b></td></tr><tr> </tr></tbody></table> </div>

<p><span class="bold">上图横坐标为样品间比较名称，纵坐标为差异蛋白数量。红柱表示上调的蛋白数量，绿色柱表示下调的蛋白数量。</span></p>

<br>
<div class="tc"><b>样品间差异蛋白注释表及Mascot分析结果</b></div>
<div style="margin:0 auto; width:900px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>样品间差异</th><th>样品间差异蛋白注释表</th><th>样品间比较 Mascot 分析结果</th></tr>
<tr><td>CK1-VS-CK2</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/2_Sample_Diff/CK1-VS-CK2.annotation.xls" target="_blank">CK1-VS-CK2.annotation.xls</a></td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/2_Sample_Diff/CK1-VS-CK2.mascot.xls" target="_blank">CK1-VS-CK2.mascot.xls</a></td></tr>
<tr><td>CK1-VS-FM1</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/2_Sample_Diff/CK1-VS-FM1.annotation.xls" target="_blank">CK1-VS-FM1.annotation.xls</a></td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/2_Sample_Diff/CK1-VS-FM1.mascot.xls" target="_blank">CK1-VS-FM1.mascot.xls</a></td></tr>
<tr><td>CK1-VS-FM2</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/2_Sample_Diff/CK1-VS-FM2.annotation.xls" target="_blank">CK1-VS-FM2.annotation.xls</a></td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/2_Sample_Diff/CK1-VS-FM2.mascot.xls" target="_blank">CK1-VS-FM2.mascot.xls</a></td></tr>
<tr><td>CK1-VS-MA1</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/2_Sample_Diff/CK1-VS-MA1.annotation.xls" target="_blank">CK1-VS-MA1.annotation.xls</a></td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/2_Sample_Diff/CK1-VS-MA1.mascot.xls" target="_blank">CK1-VS-MA1.mascot.xls</a></td></tr>
<tr><td>CK1-VS-MA2</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/2_Sample_Diff/CK1-VS-MA2.annotation.xls" target="_blank">CK1-VS-MA2.annotation.xls</a></td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/2_Sample_Diff/CK1-VS-MA2.mascot.xls" target="_blank">CK1-VS-MA2.mascot.xls</a></td></tr>
<tr><td>CK1-VS-UF1</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/2_Sample_Diff/CK1-VS-UF1.annotation.xls" target="_blank">CK1-VS-UF1.annotation.xls</a></td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/2_Sample_Diff/CK1-VS-UF1.mascot.xls" target="_blank">CK1-VS-UF1.mascot.xls</a></td></tr>
<tr><td>CK1-VS-UF2</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/2_Sample_Diff/CK1-VS-UF2.annotation.xls" target="_blank">CK1-VS-UF2.annotation.xls</a></td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/2_Sample_Diff/CK1-VS-UF2.mascot.xls" target="_blank">CK1-VS-UF2.mascot.xls</a></td></tr>
<tr><td colspan="3" style="border-bottom:2px #60EF60 solid; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60">&nbsp;</td></tr>
</tbody></table></div>

<br><hr><br>


<div style="margin:0 auto; width:1005px;"> <table style="border:0px #000000 solid;">
<tbody><tr style="border:0px #000000 solid;">

<td style="border:0px #000000 solid;"><div class="tc">
<a href="./group_diff.png" target="_blank"><img style="width:500px" src="./group_diff.png"></a>
</div></td>

<td style="border:0px #000000 solid;" rowspan="2"><div class="tc">
<div class="tc"><b>分组间差异蛋白统计表</b></div>
<div style="margin:0 auto; width:500px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>分组间比较</th><th>上调蛋白数量</th><th>下调蛋白数量</th></tr>
<tr><td>FM/CK</td><td>117</td><td>84</td></tr>
<tr><td>MA/CK</td><td>219</td><td>330</td></tr>
<tr><td>UF/CK</td><td>171</td><td>78</td></tr>
<tr><td>UF/FM</td><td>11</td><td>8</td></tr>
<tr><td colspan="3" style="border-bottom:2px #60EF60 solid; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60">依据蛋白质丰度水平，当差异倍数达到1.2倍以上，<br>且经统计检验其p-value值小于0.05时，视为差异蛋白。</td></tr>
</tbody></table></div>
</div></td>
</tr>

<tr><td><b>分组间差异蛋白数量统计图</b></td></tr><tr> </tr></tbody></table> </div>

<p><span class="bold">上图横坐标为分组间比较名称，纵坐标为差异蛋白数量。红柱表示上调的蛋白数量，绿色柱表示下调的蛋白数量。</span></p>

<br>
<div class="tc"><b>分组间差异蛋白注释表</b></div>
<div style="margin:0 auto; width:900px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>样品间差异</th><th>样品间差异蛋白注释表</th></tr>
<tr><td>CK-VS-MA</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/3_Group_Diff/CK-VS-MA.annotation.xls" target="_blank">CK-VS-MA.annotation.xls</a></td></tr>
<tr><td>CK-VS-FM</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/3_Group_Diff/CK-VS-FM.annotation.xls" target="_blank">CK-VS-FM.annotation.xls</a></td></tr>
<tr><td>CK-VS-UF</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/3_Group_Diff/CK-VS-UF.annotation.xls" target="_blank">CK-VS-UF.annotation.xls</a></td></tr>
<tr><td>FM-VS-UF</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/2_Exp_Diff/3_Group_Diff/FM-VS-UF.annotation.xls" target="_blank">FM-VS-UF.annotation.xls</a></td></tr>
<tr><td colspan="2" style="border-bottom:2px #60EF60 solid; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60">&nbsp;</td></tr>
</tbody></table></div>
<a name="diff_path_anno"></a>
<p><br></p>
<h3> （2）差异蛋白Pathway富集分析 </h3>
<br>

<p><span class="bold">Pathway 显著性富集分析方法是以 KEGG Pathway 为单位，应用超几何检验，找出与所有鉴定到蛋白背景相比，在差异蛋白中显著性富集的 Pathway。通过 Pathway 显著性富集能确定差异蛋白参与的最主要生化代谢途径和信号转导途径。</span></p>
<div style="margin:0 auto; width:805px;"> <table style="border:0px #000000 solid;"> <tbody><tr style="border:0px #000000 solid;"> <td style="border:0px #000000 solid;"><div class="tc">
<a href="./pathway.png" target="_blank"><img style="width:800px" src="./pathway.png"></a>
</div></td> </tr></tbody></table> </div>
<p><span class="bold">上图为富集分析结果的示例截图。</span></p>

<br>
<div class="tc"><b>样品间比较Pathway富集结果展示</b></div>
<div style="margin:0 auto; width:1200px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>样品间差异</th><th>KO 富集结果</th><th>KO 对应表</th><th>Pathway 对应表</th><th>KEGG 富集图</th></tr>
<tr><td>CK1-VS-CK2</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-CK2.htm" target="_blank">CK1-VS-CK2.htm</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-CK2.ko" target="_blank">CK1-VS-CK2.ko</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-CK2.path.xls" target="_blank">CK1-VS-CK2.path.xls</a></td>
<td><a href="./CK1-VS-CK2.path.png" target="_blank"><img style="width:30%" src="./CK1-VS-CK2.path.png"></a></td></tr>
<tr><td>CK1-VS-FM1</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-FM1.htm" target="_blank">CK1-VS-FM1.htm</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-FM1.ko" target="_blank">CK1-VS-FM1.ko</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-FM1.path.xls" target="_blank">CK1-VS-FM1.path.xls</a></td>
<td><a href="./CK1-VS-FM1.path.png" target="_blank"><img style="width:30%" src="./CK1-VS-FM1.path.png"></a></td></tr>
<tr><td>CK1-VS-FM2</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-FM2.htm" target="_blank">CK1-VS-FM2.htm</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-FM2.ko" target="_blank">CK1-VS-FM2.ko</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-FM2.path.xls" target="_blank">CK1-VS-FM2.path.xls</a></td>
<td><a href="./CK1-VS-FM2.path.png" target="_blank"><img style="width:30%" src="./CK1-VS-FM2.path.png"></a></td></tr>
<tr><td>CK1-VS-MA1</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-MA1.htm" target="_blank">CK1-VS-MA1.htm</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-MA1.ko" target="_blank">CK1-VS-MA1.ko</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-MA1.path.xls" target="_blank">CK1-VS-MA1.path.xls</a></td>
<td><a href="./CK1-VS-MA1.path.png" target="_blank"><img style="width:30%" src="./CK1-VS-MA1.path.png"></a></td></tr>
<tr><td>CK1-VS-MA2</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-MA2.htm" target="_blank">CK1-VS-MA2.htm</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-MA2.ko" target="_blank">CK1-VS-MA2.ko</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-MA2.path.xls" target="_blank">CK1-VS-MA2.path.xls</a></td>
<td><a href="./CK1-VS-MA2.path.png" target="_blank"><img style="width:30%" src="./CK1-VS-MA2.path.png"></a></td></tr>
<tr><td>CK1-VS-UF1</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-UF1.htm" target="_blank">CK1-VS-UF1.htm</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-UF1.ko" target="_blank">CK1-VS-UF1.ko</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-UF1.path.xls" target="_blank">CK1-VS-UF1.path.xls</a></td>
<td><a href="./CK1-VS-UF1.path.png" target="_blank"><img style="width:30%" src="./CK1-VS-UF1.path.png"></a></td></tr>
<tr><td>CK1-VS-UF2</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-UF2.htm" target="_blank">CK1-VS-UF2.htm</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-UF2.ko" target="_blank">CK1-VS-UF2.ko</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/KO/CK1-VS-UF2.path.xls" target="_blank">CK1-VS-UF2.path.xls</a></td>
<td><a href="./CK1-VS-UF2.path.png" target="_blank"><img style="width:30%" src="./CK1-VS-UF2.path.png"></a></td></tr>
<tr><td colspan="5" style="border-bottom:2px #60EF60 solid; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60">&nbsp;</td></tr>
</tbody></table></div>
<br>
<div class="tc"><b>分组间比较Pathway富集结果展示</b></div>
<div style="margin:0 auto; width:1200px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>分组间差异</th><th>KO 富集结果</th><th>KO 对应表</th><th>Pathway 对应表</th><th>KEGG 富集图</th></tr>
<tr><td>CK-VS-MA</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/KO/CK-VS-MA.htm" target="_blank">CK-VS-MA.htm</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/KO/CK-VS-MA.ko" target="_blank">CK-VS-MA.ko</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/KO/CK-VS-MA.path.xls" target="_blank">CK-VS-MA.path.xls</a></td>
<td><a href="./CK-VS-MA.path.png" target="_blank"><img style="width:30%" src="./CK-VS-MA.path.png"></a></td></tr>
<tr><td>CK-VS-FM</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/KO/CK-VS-FM.htm" target="_blank">CK-VS-FM.htm</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/KO/CK-VS-FM.ko" target="_blank">CK-VS-FM.ko</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/KO/CK-VS-FM.path.xls" target="_blank">CK-VS-FM.path.xls</a></td>
<td><a href="./CK-VS-FM.path.png" target="_blank"><img style="width:30%" src="./CK-VS-FM.path.png"></a></td></tr>
<tr><td>CK-VS-UF</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/KO/CK-VS-UF.htm" target="_blank">CK-VS-UF.htm</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/KO/CK-VS-UF.ko" target="_blank">CK-VS-UF.ko</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/KO/CK-VS-UF.path.xls" target="_blank">CK-VS-UF.path.xls</a></td>
<td><a href="./CK-VS-UF.path.png" target="_blank"><img style="width:30%" src="./CK-VS-UF.path.png"></a></td></tr>
<tr><td>FM-VS-UF</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/KO/FM-VS-UF.htm" target="_blank">FM-VS-UF.htm</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/KO/FM-VS-UF.ko" target="_blank">FM-VS-UF.ko</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/KO/FM-VS-UF.path.xls" target="_blank">FM-VS-UF.path.xls</a></td>
<td><a href="./FM-VS-UF.path.png" target="_blank"><img style="width:30%" src="./FM-VS-UF.path.png"></a></td></tr>
<tr><td colspan="5" style="border-bottom:2px #60EF60 solid; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60">&nbsp;</td></tr>
</tbody></table></div>
<br>
<a name="diff_go_ana"></a>
<p><br></p>
<h2> （3）差异蛋白GO富集分析</h2>
<br>

<p><span class="bold">GO功能显著性富集分析给出与所有鉴定到的蛋白质背景相比，差异蛋白质中显著富集的GO功能条目，从而给出差异蛋白质与哪些生物学功能显著相关。</span></p>
<p><span class="bold">当蛋白的丰度比即差异倍数达到1.5倍以上，且经统计检验其p-value值小于0.05时，视该蛋白为不同样品间的差异蛋白。</span></p>
<p><span class="bold">该分析首先把所有差异蛋白质向 Gene Ontology 数据库（http://www.geneontology.org/）的各个 term 映射，计算每个 term 的蛋白质数目，然后应用超几何检验，找出与所有蛋白质背景相比，在差异蛋白质中显著富集的 GO 条目。其计算公式为：</span></p>
<div style="margin:0 auto; width:805px;"> <table style="border:0px #000000 solid;"> <tbody><tr style="border:0px #000000 solid;"> <td style="border:0px #000000 solid;"><div class="tc">
<a href="./p_value.png" target="_blank"><img style="width:300px" src="./p_value.png"></a>
</div></td> </tr></tbody></table> </div>
<p><span class="bold">其中N为所有蛋白中具有GO 注释信息的蛋白数目，n为N中差异蛋白的数目，M为所有蛋白中注释到某个GO 条目的蛋白数目，m为注释到某个GO 条目的差异蛋白数目。</span>/p&lt;&gt;
</p><p><span class="bold">计算得到P-value值，以P-value≤0.05为阈值，满足此条件的GO term定义为在差异蛋白质中显著富集的GO term。通过GO显著性分析能确定差异蛋白性行使的主要生物学功能。</span></p>
<div style="margin:0 auto; width:805px;"> <table style="border:0px #000000 solid;"> <tbody><tr style="border:0px #000000 solid;"> <td style="border:0px #000000 solid;"><div class="tc">
<a href="./tab01.jpg" target="_blank"><img style="width:800px" src="./tab01.jpg"></a>
</div></td> </tr></tbody></table> </div>
<p><span class="bold">上图为 GO 富集分析结果的截图。Gene Onltolgy term 为 GO 条目。Cluster frequency 为注释到该条目的差异蛋白与注释到所有 GO 条目的差异蛋白的数量和比值。Protein frequency of use为注释到该条目的所有蛋白与注释到所有 GO 条目的所有蛋白的数量和比值。P-value 为通过超几何检验计算得到 P-value值。</span></p>
<br>
<div class="tc"><b>样品间比较 GO 富集结果展示</b></div>
<div style="margin:0 auto; width:1200px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>样品间差异</th><th>GO 细胞组分(C)<br>富集结果</th><th>GO 分子功能(F)<br>富集结果</th><th>GO 生物过程(P)<br>富集结果</th><th>GO 注释分类表</th><th>GO 功能分类图</th></tr>
<tr><td>CK1-VS-CK2</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-CK2_C.html" target="_blank">CK1-VS-CK2_C.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-CK2_F.html" target="_blank">CK1-VS-CK2_F.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-CK2_P.html" target="_blank">CK1-VS-CK2_P.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-CK2.go.class.xls" target="_blank">CK1-VS-CK2.go.class.xls</a></td>
<td><a href="./CK1-VS-CK2.go.class.png" target="_blank"><img style="width:30%" src="./CK1-VS-CK2.go.class.png"></a></td></tr>
<tr><td>CK1-VS-FM1</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-FM1_C.html" target="_blank">CK1-VS-FM1_C.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-FM1_F.html" target="_blank">CK1-VS-FM1_F.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-FM1_P.html" target="_blank">CK1-VS-FM1_P.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-FM1.go.class.xls" target="_blank">CK1-VS-FM1.go.class.xls</a></td>
<td><a href="./CK1-VS-FM1.go.class.png" target="_blank"><img style="width:30%" src="./CK1-VS-FM1.go.class.png"></a></td></tr>
<tr><td>CK1-VS-FM2</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-FM2_C.html" target="_blank">CK1-VS-FM2_C.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-FM2_F.html" target="_blank">CK1-VS-FM2_F.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-FM2_P.html" target="_blank">CK1-VS-FM2_P.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-FM2.go.class.xls" target="_blank">CK1-VS-FM2.go.class.xls</a></td>
<td><a href="./CK1-VS-FM2.go.class.png" target="_blank"><img style="width:30%" src="./CK1-VS-FM2.go.class.png"></a></td></tr>
<tr><td>CK1-VS-MA1</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-MA1_C.html" target="_blank">CK1-VS-MA1_C.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-MA1_F.html" target="_blank">CK1-VS-MA1_F.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-MA1_P.html" target="_blank">CK1-VS-MA1_P.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-MA1.go.class.xls" target="_blank">CK1-VS-MA1.go.class.xls</a></td>
<td><a href="./CK1-VS-MA1.go.class.png" target="_blank"><img style="width:30%" src="./CK1-VS-MA1.go.class.png"></a></td></tr>
<tr><td>CK1-VS-MA2</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-MA2_C.html" target="_blank">CK1-VS-MA2_C.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-MA2_F.html" target="_blank">CK1-VS-MA2_F.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-MA2_P.html" target="_blank">CK1-VS-MA2_P.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-MA2.go.class.xls" target="_blank">CK1-VS-MA2.go.class.xls</a></td>
<td><a href="./CK1-VS-MA2.go.class.png" target="_blank"><img style="width:30%" src="./CK1-VS-MA2.go.class.png"></a></td></tr>
<tr><td>CK1-VS-UF1</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-UF1_C.html" target="_blank">CK1-VS-UF1_C.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-UF1_F.html" target="_blank">CK1-VS-UF1_F.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-UF1_P.html" target="_blank">CK1-VS-UF1_P.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-UF1.go.class.xls" target="_blank">CK1-VS-UF1.go.class.xls</a></td>
<td><a href="./CK1-VS-UF1.go.class.png" target="_blank"><img style="width:30%" src="./CK1-VS-UF1.go.class.png"></a></td></tr>
<tr><td>CK1-VS-UF2</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-UF2_C.html" target="_blank">CK1-VS-UF2_C.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-UF2_F.html" target="_blank">CK1-VS-UF2_F.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-UF2_P.html" target="_blank">CK1-VS-UF2_P.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/4_Sample_Diff_Enrich/All/GO/CK1-VS-UF2.go.class.xls" target="_blank">CK1-VS-UF2.go.class.xls</a></td>
<td><a href="./CK1-VS-UF2.go.class.png" target="_blank"><img style="width:30%" src="./CK1-VS-UF2.go.class.png"></a></td></tr>
<tr><td colspan="6" style="border-bottom:2px #60EF60 solid; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60">&nbsp;</td></tr>
</tbody></table></div>
<br>
<div class="tc"><b>分组间比较 GO 富集结果展示</b></div>
<div style="margin:0 auto; width:1200px;">
<table align="center" style="border-left:2px #AAAAAA solid; border-right:2px #AAAAAA solid; border-top:2px #AAAAAA solid;">
<tbody><tr style="border-bottom:2px #AAAAAA solid;"><th>分组间差异</th><th>GO 细胞组分(C)<br>富集结果</th><th>GO 分子功能(F)<br>富集结果</th><th>GO 生物过程(P)<br>富集结果</th><th>GO 注释分类表</th><th>GO 功能分类图</th></tr>
<tr><td>CK-VS-MA</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/GO/CK-VS-MA_C.html" target="_blank">CK-VS-MA_C.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/GO/CK-VS-MA_F.html" target="_blank">CK-VS-MA_F.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/GO/CK-VS-MA_P.html" target="_blank">CK-VS-MA_P.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/GO/CK-VS-MA.go.class.xls" target="_blank">CK-VS-MA.go.class.xls</a></td>
<td><a href="./CK-VS-MA.go.class.png" target="_blank"><img style="width:30%" src="./CK-VS-MA.go.class.png"></a></td></tr>
<tr><td>CK-VS-FM</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/GO/CK-VS-FM_C.html" target="_blank">CK-VS-FM_C.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/GO/CK-VS-FM_F.html" target="_blank">CK-VS-FM_F.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/GO/CK-VS-FM_P.html" target="_blank">CK-VS-FM_P.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/GO/CK-VS-FM.go.class.xls" target="_blank">CK-VS-FM.go.class.xls</a></td>
<td><a href="./CK-VS-FM.go.class.png" target="_blank"><img style="width:30%" src="./CK-VS-FM.go.class.png"></a></td></tr>
<tr><td>CK-VS-UF</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/GO/CK-VS-UF_C.html" target="_blank">CK-VS-UF_C.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/GO/CK-VS-UF_F.html" target="_blank">CK-VS-UF_F.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/GO/CK-VS-UF_P.html" target="_blank">CK-VS-UF_P.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/GO/CK-VS-UF.go.class.xls" target="_blank">CK-VS-UF.go.class.xls</a></td>
<td><a href="./CK-VS-UF.go.class.png" target="_blank"><img style="width:30%" src="./CK-VS-UF.go.class.png"></a></td></tr>
<tr><td>FM-VS-UF</td><td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/GO/FM-VS-UF_C.html" target="_blank">FM-VS-UF_C.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/GO/FM-VS-UF_F.html" target="_blank">FM-VS-UF_F.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/GO/FM-VS-UF_P.html" target="_blank">FM-VS-UF_P.html</a></td>
<td><a href="http://www.biotrainee.com:8081/jmzeng/html_report/d/e/e/p/i/n/iTRAQ_Result/3_Annotation/5_Group_Diff_Enrich/All/GO/FM-VS-UF.go.class.xls" target="_blank">FM-VS-UF.go.class.xls</a></td>
<td><a href="./FM-VS-UF.go.class.png" target="_blank"><img style="width:30%" src="./FM-VS-UF.go.class.png"></a></td></tr>
<tr><td colspan="6" style="border-bottom:2px #60EF60 solid; border-top:solid #AAAAAA 2px; text-align:left; font-size:80%; background-color:#60EF60">&nbsp;</td></tr>
</tbody></table></div>
<br>
<a name="data"></a>
<p><br></p>
<div class="h0"><li>四、文件格式说明</li></div>
<br>

<a name="file"></a>
<p><br></p>
<h2> 1）文件下载及解压</h2>
<p><span class="bold">所有文件被压缩成*.tar.gz格式，可以采取如下解压方式：</span></p>
<p><span class="bold">Unix/Linux用户： tar Czxvf *.tar.gz or gzip Cd *.gz。</span></p>
<p><span class="bold">Windows用户：推荐使用winRAR进行。</span></p>
<p><span class="bold">Mac 用户： shell: tar Czxvf *.tar.gz。</span></p>

<br>

<a name="pro_help"></a>
<p><br></p>
<h2> 2）蛋白鉴定及定量</h2>
<div style="margin:0 auto; width:805px;"> <table style="border:0px #000000 solid;"> <tbody><tr style="border:0px #000000 solid;"> <td style="border:0px #000000 solid;"><div class="tc">
<a href="./pro_xls.png" target="_blank"><img style="width:800px" src="./pro_xls.png"></a>
</div></td> </tr></tbody></table> </div>
<p><span class="bold">上图为iTRAQ定量分析报告中的*_detail.xlsx中表格信息，每列的说明信息如下表：</span></p>
<table style="width:900px">
<tbody><tr><th>列名</th><th>描述</th></tr>
<tr><td>GroupId</td><td>蛋白质分组号，分组号相同的蛋白共享肽段及定量信息</td></tr>
<tr><td>Hit_number</td><td>Mascot搜索结果中的分组号，为不连续编号</td></tr>
<tr><td>Accession</td><td>蛋白质accession number</td></tr>
<tr><td>Description</td><td>蛋白质描述信息</td></tr>
<tr><td>Score</td><td>蛋白质的鉴定得分</td></tr>
<tr><td>Mass</td><td>蛋白质质量</td></tr>
<tr><td>Cov</td><td>蛋白的序列覆盖度情况</td></tr>
<tr><td>Seq</td><td>蛋白序列</td></tr>
<tr><td>SameSets</td><td>与该蛋白鉴定到完全相同肽段的蛋白的数量</td></tr>
<tr><td>Sepectra number</td><td>匹配到该蛋白的谱的数量</td></tr>
<tr><td>Unique peptide number</td><td>匹配到该蛋白的unique肽段数量</td></tr>
<tr><td>Peptide</td><td>匹配到该蛋白的肽段序列数量</td></tr>
<tr><td>Unique Peptide</td><td>匹配到该蛋白的unique肽段序列数量</td></tr>
<tr><td>118/114</td><td>该蛋白在两个样品间表达量的比值，即差异倍数</td></tr>
<tr><td>Quant Number</td><td>用于该蛋白定量的肽段数量</td></tr>
<tr><td>Sig</td><td>统计检验其表达量与1是否有统计学差异，1表示有统计学差异</td></tr>
<tr><td>……</td><td>……</td></tr>
<tr><td>Query</td><td>蛋白的肽段谱图编号</td></tr>
<tr><td>Isunique</td><td>表示该谱图是否是unique肽段，1为unique，0为非unique</td></tr>
<tr><td>Exp_mz</td><td>实验测得的肽段的质荷比值</td></tr>
<tr><td>Exp_mr</td><td>实验测得的肽段的分子量</td></tr>
<tr><td>Exp_z</td><td>肽段的电荷量</td></tr>
<tr><td>Calc_mr</td><td>理论计算得到的肽段分子量</td></tr>
<tr><td>Delta</td><td>肽段的实验分子量与理论分子量的差值</td></tr>
<tr><td>Start</td><td>该谱匹配到的肽段在蛋白序列中的起始位置</td></tr>
<tr><td>End</td><td>该谱匹配到的肽段在蛋白序列中的结束位置</td></tr>
<tr><td>Miss</td><td>肽段酶切时的漏切个数</td></tr>
<tr><td>Score</td><td>肽段得分</td></tr>
<tr><td>Ident</td><td>肽段的鉴定得分阈值</td></tr>
<tr><td>Before</td><td>位于肽段之前的第一个氨基酸</td></tr>
<tr><td>Pep_seq</td><td>肽段序列</td></tr>
<tr><td>After</td><td>位于肽段之后的第一个氨基酸</td></tr>
<tr><td>Pep_var_mod</td><td>肽段的修饰方式</td></tr>
</tbody></table>
<br>

<a name="go_ana_help"></a>
<p><br></p>
<h2> 3）GO功能注释</h2>
<br>
<a name="help_go_xls"></a>
<p><span class="bold">GO 分类表每列的说明信息如下：</span></p>
<table style="width:900px">
<tbody><tr><th>列名</th><th>描述</th></tr>
<tr><td>Ontology</td><td>GO本体的类别<br>(biological_process 或 cellular_component 或 molecular_function)</td></tr>
<tr><td>Class</td><td>GO条目</td></tr>
<tr><td>number_of_*</td><td>注释到各GO条目中的蛋白质数量</td></tr>
<tr><td>Proteins_of_*</td><td>注释到各GO条目蛋白质ID</td></tr>
</tbody></table>

<a name="cog_anno_help"></a>
<p><br></p>
<h2> 4）COG/KOG功能注释</h2>
<br>

<a name="help_cog_class"></a>
<p><span class="bold">COG/KOG 分类表每列的说明信息如下：</span></p>
<table style="width:900px">
<tbody><tr><th>列名</th><th>描述</th></tr>
<tr><td>Code</td><td>COG/KOG 功能代号</td></tr>
<tr><td>Functional-Categories</td><td>COG/KOG 功能分类</td></tr>
<tr><td>Protein-Number</td><td>蛋白质 在各 COG/KOG 功能分类的数量</td></tr>
<tr><td>Proteins</td><td>蛋白质 在各 COG/KOG 功能分类的序列ID</td></tr>
</tbody></table>
<br>

<a name="help_cog_annot"></a>
<p><span class="bold">COG/KOG 注释表每列的说明信息如下：</span></p>
<table style="width:900px">
<tbody><tr><th>列名</th><th>描述</th></tr>
<tr><td>Protein</td><td>蛋白质 序列的名称</td></tr>
<tr><td>Protein-or-Domain</td><td>比对上的蛋白或结构域</td></tr>
<tr><td>Score</td><td>blast比对的得分</td></tr>
<tr><td>E-Value</td><td>blast比对的Evalue</td></tr>
<tr><td>COG/KOG-ID</td><td>COG/KOG 的ID</td></tr>
<tr><td>Function-Description</td><td>功能描述</td></tr>
<tr><td>Code</td><td>COG/KOG 功能代号</td></tr>
<tr><td>Functional-Categories</td><td>COG/KOG 功能分类</td></tr>
</tbody></table>


<a name="path_anno_help"></a>
<p><br></p>
<h2> 5）Pathway功能注释</h2>
<br>

<a name="help_pathway_list"></a>
<p><span class="bold">Pathway 表每列的说明信息如下：</span></p>
<table style="width:900px">
<tbody><tr><th>列名</th><th>描述</th></tr>
<tr><td>KEGG_A_Class</td><td>KEGG A 级分类</td></tr>
<tr><td>KEGG_B_Class</td><td>KEGG B 级分类</td></tr>
<tr><td>Pathway</td><td>通路名 （KEGG C 级分类）</td></tr>
<tr><td>Count</td><td>注释到该通路的蛋白质的数目</td></tr>
<tr><td>Pathway ID</td><td>KEGG数据库中的Pathway ID</td></tr>
<tr><td>Proteins</td><td>注释到该Pathway的蛋白</td></tr>
<tr><td>KOs</td><td>属于该Pathway的KEGG Orthology</td></tr>
</tbody></table>

<a name="help_path_htm"></a>
<p><span class="bold"> *.htm 为蛋白的Pathway分析结果文件，说明信息如下：</span></p>
<table style="width:900px">
<tbody><tr><th>列名</th><th>描述</th></tr>
<tr><td>Pathway</td><td>通路名，点击可看到注释到该通路的各蛋白质情况及Pathway图</td></tr>
<tr><td>Proteins with pathway annotation</td><td>鉴定的蛋白质注释到该通路的数目及比例</td></tr>
<tr><td>Pathway ID</td><td>KEGG数据库中的Pathway ID</td></tr>
</tbody></table>


<a name="diff_go_ana_help"></a>
<p><br></p>
<h2> 6）差异蛋白的GO富集分析</h2>
<br>

<p><span class="bold"> GOView.html为差异蛋白质GO功能富集分析结果报告，左侧导航页面分别描述了如下信息：</span></p>
<table style="width:900px">
<tbody><tr><th>文件名</th><th>描述</th></tr>
<tr><td>*_diff_C</td><td>其中差异蛋白质的Go-CellularComponent分析结果</td></tr>
<tr><td>*_diff_F</td><td>其中差异蛋白质的Go-MolecularFunction分析结果</td></tr>
<tr><td>*_diff_P</td><td>其中差异蛋白质的Go-BiologicalProcess分析结果</td></tr>
</tbody></table>

<br>

<p><span class="bold"> 网页内右侧框架内表格说明信息如下：</span></p>
<table style="width:900px">
<tbody><tr><th>列名</th><th>描述</th></tr>
<tr><td>Gene Ontology term</td><td>GO term名</td></tr>
<tr><td>Cluster frequency</td><td>注释该GO term的差异蛋白质与注释到所有GO term的差异蛋白质比值</td></tr>
<tr><td>Protein frequency of use</td><td>注释该GO term 的所有蛋白质与注释到所有GO term的所有蛋白质的比值</td></tr>
<tr><td>Corrected P-value</td><td>通过超几何检验计算得到P-value值，当该值&lt;0.05时，定义为在差异蛋白质中显著富集的GO term</td></tr>
</tbody></table>


<a name="diff_path_anno_help"></a>
<p><br></p>
<h2> 7）差异蛋白的Pathway富集分析</h2>
<br>

<p><span class="bold"> *.htm 为差异蛋白的Pathway富集分析结果文件，说明信息如下：</span></p>
<table style="width:900px">
<tbody><tr><th>列名</th><th>描述</th></tr>
<tr><td>Pathway</td><td>通路名，点击可看到注释到该通路的各蛋白质情况及Pathway图</td></tr>
<tr><td>Diff Proteins with pathway annotation</td><td>鉴定的差异蛋白质注释到该通路的数目及比例</td></tr>
<tr><td>Proteins with pathway annotation</td><td>鉴定的蛋白质注释到该通路的数目及比例</td></tr>
<tr><td>P-value</td><td>通过超几何检验计算得到p-value值，当该值&lt;0.05时，定义为在差异蛋白质中显著富集的Pathway</td></tr>
<tr><td>Pathway ID</td><td>KEGG数据库中的Pathway ID</td></tr>
</tbody></table>
<br>

<br><hr> <!-- ---------------------------------------------------------------------------------------------------------------------------- -->
<a name="literature"></a>
<p><br></p>
<div class="h0"><li>六、参考文献</li></div>

<br>

<ul>

	<li><a name="cite1"></a>1. R. S. Johnson and J. A. Taylor (2002). <font color="#00688B">Searching sequence databases via de novo peptide sequencing by tandem mass spectrometry.</font> <b><i>Mol. Biotechnology.</i></b> Vol. 22, No. 3. (November 2002), pp. 301-315.</li>
	<li><a name="cite2"></a>2. J. A. Taylor and R. S. Johnson (2000). <font color="#00688B">Implementation and uses of automated de novo peptide sequencing by tandem mass spectrometry.</font> <b><i>Anal. Chem.</i></b> 73:2594-2604. </li>
	<li><a name="cite3"></a>3. R. S. Johnson and J. A. Taylor (2000). <font color="#00688B">Searching sequence databases via de novo peptide sequencing by tandem mass spectrometry.</font> <b><i>Methods in Molecular Biology series, Humana Press.</i></b></li>
	<li><a name="cite4"></a>4. J. A. Taylor and R. S. Johnson (1997). <font color="#00688B">Searching sequence databases via de novo peptide sequencing by tandem mass spectrometry.</font> <b><i>Rapid Comm. Mass Spec.</i></b> 11:1067-1075.</li>
	<li><a name="cite5"></a>5. Frank, A.M. <font color="#00688B">Predicting Intensity Ranks of Peptide Fragment Ions.</font> <b><i>J. Proteome Research.</i></b> 8:2226-2240, 2009.</li>
	<li><a name="cite6"></a>6. Frank, A.M. <font color="#00688B">Ranking-Based Scoring Models for Peptide-Spectrum Matches.</font> <b><i>J. Proteome Research.</i></b> 8:2241-2252, 2009.</li>
	<li><a name="cite7"></a>7. Ari M. Frank, Mikhail M. <i>et al.</i> <font color="#00688B">De Novo Peptide Sequencing and Identification with Precision Mass Spectrometry.</font> <b><i>J. Proteome Res. </i></b> 6:114-123, 2007.</li>
	<li><a name="cite8"></a>8. Frank, A. and Pevzner, P. <font color="#00688B">PepNovo: De Novo Peptide Sequencing via Probabilistic Network Modeling.</font> <b><i>Analytical Chemistry.</i></b> 77:964-973, 2005.</li>
	<li><a name="cite9"></a>9. Frank A, Tanner S, Bafna V, Pevzner P.. <font color="#00688B">Peptide sequence tags for fast database search in mass-spectrometry.</font> <b><i>J Proteome Res.</i></b> 2005 Jul-Aug;4(4):1287-95.</li>
	<li><a name="cite10"></a>10. Ari Frank, Stephen Tanner, and Pavel Pevzner. <font color="#00688B">Peptide Sequence Tags for Fast Database Search in Mass Spectrometry.</font> <b><i>Conference on Research in Computational Molecular Biology (RECOMB) 2005.</i></b></li>
</ul>
<br>

<hr> 

<br><br><br><br><br><br><br><br><br><br><br><br>

</div>

  
</body></html>